src/StatusQ/Controls/StatusSeedPhraseInput.qml

```diff
@@ -40,24 +40,58 @@ import StatusQ.Core.Theme 0.1
        }
    \endqml
 
+   \image status_seed_phrase_input.png
+
    For a list of components available see StatusQ.
 */
 
 Item {
     id: root
     width: 162
     height: 44
+    /*!
+        \qmlproperty textEdit
+        This property is an alias to the StatusInput's textEdit property.
+    */
     property alias textEdit: seedWordInput
+    /*!
+        \qmlproperty text
+        This property is an alias to the StatusInput's text property.
+    */
     property alias text: seedWordInput.text
+    /*!
+        \qmlproperty text
+        This property sets the StatusInput's left component's text.
+    */
     property string leftComponentText: ""
+    /*!
+        \qmlproperty leftComponentText
+        This property holds the seed words list.
+    */
     property ListModel inputList: ListModel { }
+    /*!
+        \qmlproperty inputList
+        This property holds the filtered words list based on the user's
+        input text.
+    */
     property ListModel filteredList: ListModel { }
+    /*!
+        \qmlproperty filteredList
+        This signal is emitted when the user selects a word from
+        the suggestions list, either by clicking on it or by completing
+        typing 4 charactersnd passes as a parameter the selected word.
+        The corresponding handler is \c onDoneInsertingWord
+    */
     signal doneInsertingWord(string word)
-    /*
-        This signal is emitted when the user pressed a keyboard key and passes as a
+    /*!
```

Comment:
If you use a specific tag, like for properties but for signals, `\qmlsignal`, that classifies them in a certain way in the docu file.

Reply:
done